optuna/study/study.py

```diff
@@ -1007,7 +1007,7 @@ def objective(trial):
         """
         if len(self.directions) != len(metric_names):
             raise ValueError(
-                "The number of objectives must match thhe length of the metric names."
+                "The number of objectives must match the length of the metric names."
```

Comment:
@wouterzwerink Thank you for your PR. The change seems obvious, but the CI failed. Could you follow the reviewdog's comment, please?

Reply:
Ha, seems like it fits within the line limit after removing this one character

Reply:
Absolutely. Thanks for the update.